Cargo.toml

```diff
@@ -44,7 +44,7 @@ serde_test = "1.0.130"
 debug = true
 
 [package.metadata.docs.rs]
-all-features = true
+features = ["serde", "weak", "internal-test-strategies", "experimental-strategies"]
```

Comment:
You're right about excluding the experimental-thread-local. But I think we may also want to exclude internal-test-strategies and experimental-strategies when at it (the internal test strategies are hidden from docs and „don't really exist“, and there are currently no existing experimental strategies present anyway)

Reply:
Updated to include only `serde` and `weak` features.